auditbeat/module/auditd/config.go

```diff
@@ -229,9 +228,9 @@ func readRules(reader io.Reader, source string, knownRules ruleSet, log *logp.Lo
 
 	if len(errs) != 0 {
 		if log == nil {
-			return nil, fmt.Errorf("failed loading rules: %w", errs.Err())
+			return nil, fmt.Errorf("failed loading rules: %w", errors.Join(errs...))
 		}
-		log.Warnf("errors loading rules: %v", errs.Err())
+		log.Warnf("errors loading rules: %v", errors.Join(errs...))
```

Comment:
```suggestion
		log.Warnf("errors loading rules: %w", errors.Join(errs...))
```

Reply:
@efd6, I think this one needs to remain as `%v`. Can you double check. I thought this delegates to sprintf which won't accept `%w`.

Reply:
You are correct. zap uses printf, not errorf.